packages/types/src/schemas/schemas.ts

```diff
@@ -59420,172 +59420,93 @@ export const DataTablePreviewRequestSchema = {
 } 
 
 export const ParamsSchema = {
```

Comment:
So there's something that's been going wrong for a little while with the schemas being generated in this file. We manually parse over the files within the types directory to control which types are generated as schemas. However, if there are naming clashes of individual types there, then those clashes cause the schemas built in this file to clash.

In order to avoid this we should probably give all the types in that directory non-clashing names (we do this currently by renaming as export). But that's a little out of scope for this PR.

Reply:
Oh for the schemas, yeah I see. Maybe the schema generator can wrap these up as well somehow.

Reply:
Yeah, we'll have to come up with something. I like the nested types shape, but I dunno how much control we have with the schema generator...